index.js

```diff
@@ -32,128 +29,53 @@ const panel = require('sdk/panel').Panel({
+        const panel = panelUtils.getPanel();
+        if (panel.isShowing) {
+          panelUtils.redraw();
+        }
+      });
     }
-    panel.port.emit('set-video', {domain: '', src: ''});
-    panel.hide();
-  } else if (title === 'close') {
-    panel.port.emit('set-video', {domain: '', src: ''});
-    panel.hide();
-  } else if (title === 'minimize') {
-    adjustHeight(dimensions.minimizedHeight);
-  } else if (title === 'maximize') {
-    adjustHeight(dimensions.height);
-  } else if (title === 'metrics-event') {
-    sendMetricsData(opts, panel);
-  }
-});
-
-function getPageUrl(domain, id, time) {
-  let url;
-  if (domain.indexOf('youtube') > -1) {
-    url = `https://youtube.com/watch?v=${id}&t=${Math.floor(time)}`;
-  } else if (domain.indexOf('vimeo') > -1) {
-    const min = Math.floor(time / 60);
-    const sec = Math.floor(time - min * 60);
-    url = `https://vimeo.com/${id}#t=${min}m${sec}s`;
-  } else if (domain.indexOf('vine') > -1) {
-    url = `https://vine.co/v/${id}`;
-  }
-
-  return url;
-}
-
-// handle browser resizing
-pageMod.PageMod({
-  include: '*',
-  contentScriptFile: './resize-listener.js',
-  onAttach: function(worker) {
-    worker.port.on('resized', function() {
-      if (panel.isShowing) {
-        redrawPanel();
-      }
-    });
-  }
-});
-
-// add launch icon to video embeds
-pageMod.PageMod({
-  include: '*',
-  contentStyleFile: './icon-overlay.css',
-  contentScriptFile: './icon-overlay.js',
-  onAttach: function(worker) {
-    worker.port.on('launch', function(opts) {
-      if (opts.domain.indexOf('youtube.com') > -1) {
-        opts.getUrlFn = getYouTubeUrl;
-        sendMetricsData({
-          object: 'overlay_icon',
-          method: 'launch',
-          domain: opts.domain
-        }, panel);
-        launchVideo(opts, panel);
-      } else if (opts.domain.indexOf('vimeo.com')  > -1) {
-        opts.getUrlFn = getVimeoUrl;
-        sendMetricsData({
-          object: 'overlay_icon',
-          method: 'launch',
-          domain: opts.domain
-        }, panel);
-        launchVideo(opts, panel);
-      }
-    });
+  });
+
+  // add launch icon to video embeds
+  pageMod.PageMod({
+    include: '*',
+    contentStyleFile: './icon-overlay.css?cachebust=' + Date.now(),
+    contentScriptFile: './icon-overlay.js?cachebust=' + Date.now(),
+    onAttach: function(worker) {
+      worker.port.on('launch', function(opts) {
+        if (opts.domain.indexOf('youtube.com') > -1) {
+          opts.getUrlFn = getYouTubeUrl;
+          sendMetricsData({
+            object: 'overlay_icon',
+            method: 'launch',
+            domain: opts.domain
+          });
+          launchVideo(opts);
+        } else if (opts.domain.indexOf('vimeo.com')  > -1) {
+          opts.getUrlFn = getVimeoUrl;
+          sendMetricsData({
+            object: 'overlay_icon',
+            method: 'launch',
+            domain: opts.domain
+          });
+          launchVideo(opts);
+        }
```

Comment:
This code was just reshuffled, not added. Good question, though. Maybe file a bug?